syntax = "proto3";

import "google/protobuf/descriptor.proto";

option java_package = "com.ibm.gwhisper.protodoc";

extend google.protobuf.MethodOptions {
    string rpc_doc = 51009;
}

extend google.protobuf.ServiceOptions {
    string service_doc = 51000;
}

extend google.protobuf.MessageOptions {
    string message_doc = 51001;
}

extend google.protobuf.FieldOptions {
    string field_doc = 51002;
}


